.github/CODEOWNERS

```diff
@@ -10,18 +10,19 @@
 # This file
 /.github/CODEOWNERS @edolstra
 
-# Boostraping and core infra
-/pkgs/stdenv                    @edolstra
-/pkgs/build-support/cc-wrapper  @edolstra
-
 # Libraries
 /lib                        @edolstra @nbp
+/lib/systems                @edolstra @nbp @ericson2314
 
 # Nixpkgs Internals
-/default.nix                @nbp
-/pkgs/top-level/default.nix @nbp
-/pkgs/top-level/impure.nix  @nbp
-/pkgs/top-level/stage.nix   @nbp
+/default.nix                          @nbp
+/pkgs/top-level/default.nix           @nbp @Ericson2314
+/pkgs/top-level/impure.nix            @nbp @Ericson2314
+/pkgs/top-level/stage.nix             @nbp @Ericson2314
+/pkgs/stdenv                          @edolstra
+/pkgs/build-support/cc-wrapper        @edolstra @Ericson2314
+/pkgs/build-support/bintools-wrapper  @edolstra @Ericson2314
```

Comment:
The PR of mine making this was just merged, and `binutils-wrapper` just as important as `cc-wrapper` (from which it was factored out of) and so everyone owning `cc-wrapper` should own it too.